ngraph_onnx/onnx_importer/ops_bridge/opset_01.py

```diff
@@ -606,11 +592,27 @@ def ConvTranspose(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]
 
 def Pad(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
     """Add padding to the input tensor."""
+    # Oprator set version 1
+    paddings = onnx_node.get_attribute_value('paddings')
+    # Operator set version >= 2
     pads = onnx_node.get_attribute_value('pads')
+
+    pads = pads if pads is not None else paddings
+    if pads is None:
+        raise ValueError('Pad node (s%): paddings attribute is required.', onnx_node.name)
```

Comment:
Change `paddings attribute is required` -> `pads attribute is required`. This applies in more cases.

Reply:
Fixed